psst/psst.py

```diff
@@ -13,6 +13,7 @@ def __init__(self):
 
 @click.group(no_args_is_help=True)
 @pass_config
+

```

Comment:
@iversond Can you kill this white space? Otherwise looks good.

Reply:
Done.